spec/swagger_schemas/models/participants/enrolment.rb

```diff
@@ -0,0 +1,164 @@
+      type: :string,
+      example: "2021",
+    },
+    training_status: {
+      description: "The training status of this participant",
+      type: :string,
+      example: "active",
+      enum: %w[active deferred withdrawn]
+    },
+    participant_status: {
+      description: "Replaces the old status field. Indicates if a SIT has advised DfE of a transfer or a withdrawal of the participant from the school",
+      type: :string,
+      enum: %w[active joining leaving left withdrawn],
+      example: "active"
+    },
+    eligible_for_funding: {
+      description: "Indicates whether this participant is eligible to receive DfE funded induction",
+      type: :boolean,
+      example: true,
+      nullable: true
+    },
+    pupil_premium_uplift: {
+      description: "Indicates whether this participant qualifies for an uplift payment due to pupil premium",
+      type: :boolean,
+      example: true
+    },
+    sparsity_uplift: {
+      description: "Indicates whether this participant qualifies for an uplift payment due to sparsity",
+      type: :boolean,
+      example: true
+    },
+    schedule_identifier: {
+      description: "The schedule of the participant",
+      type: :string,
+      enum: Schedule.identifiers.keys,
+      example: "ecf-standard-january"
+    },
+    delivery_partner_id: {
+      description: "Unique ID of the delivery partner associated with the participant",
+      type: :string,
+      format: :uuid,
+      example: "42a9ef2f-9059-400a-92ff-4830a629d0c5"
+    },
+    withdrawal: {
+      description: "The details of a participant withdrawal",
+      type: :object,
+      required: %I[reason date],
+      properties: {
+        reason: {
+          description: "The reason a participant was withdrawn",
+          type: :string,
+          enum: TrainingPeriod.withdrawal_reasons.keys.map(&:dasherize),
+          example: "moved-school"
+        },
+        date: {
+          description: "The date and time the participant was withdrawn",
+          type: :string,
+          format: "date-time",
+          example: "2021-05-31T02:22:32.000Z"
+        }
+      }
+    },
+    deferral: {
+      description: "The details of a participant deferral",
+      type: :object,
+      required: %I[reason date],
+      properties: {
+        reason: {
+          description: "The reason a participant was deferred",
+          type: :string,
+          enum: TrainingPeriod.deferral_reasons.keys.map(&:dasherize),
+          example: "career-break"
+        },
+        date: {
+          description: "The date and time the participant was deferred",
+          type: :string,
+          format: "date-time",
+          example: "2021-05-31T02:22:32.000Z"
+        }
+      }
+    },
+    created_at: {
+      description: "The date and time the participant was created",
+      type: :string,
+      format: "date-time",
+      example: "2023-01-01T00:00:00Z"
+    },
+    induction_end_date: {
```

Comment:
we have a new field in RECT (not in ECF1) called `overall_induction_start_date`, can we add it please?